scripts/database/pg/migrations/020.do.add-constraint-unique-eventbrite-event-id.sql

```diff
@@ -0,0 +1,9 @@
+DO $$
+  BEGIN
+    BEGIN
+        ALTER TABLE cd_events ADD UNIQUE (eventbrite_id);
+    EXCEPTION
+        WHEN duplicate_column THEN RAISE NOTICE 'constraint eventbrite_id already exists in cd_events.';
```

Comment:
Is that really the exception we should be catching? Surely it's something other than duplicate_column for a failure adding a constraint?

Reply:
krrrrrrr; good catch